src/Validations/trainers.jsx

```diff
@@ -2,58 +2,69 @@ import Joi from 'joi';
 
 const trainerValidation = Joi.object({
   firstName: Joi.string()
-    .regex(/^[A-Za-z]+\s?[A-Za-z]+$/)
     .trim()
+    .regex(/^[A-Za-z]+\s?[A-Za-z]*$/)
     .min(3)
     .max(25)
-    .required()
     .messages({
-      'string.pattern.base': 'Name must have only letters',
-      'any.required': 'Name is required',
-      'string.empty': 'Name is required.'
+      'any.required': 'First Name is required',
```

Comment:
It might be more consistent to match the validation message ('First Name' is required) with the tag label ('Name').